app/views/challenges/drawing.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= render partial: 'challenges/header', cache: true %>
+
+<div id='challenge-drawing'>
+  <%= @challenge.drawing.html_safe if @challenge.drawing %>
```

Comment:
E.g. put the container, row, col-md stuff here like we do in all of our other partials.